docs/application/native/guides/text-input/tts.md

```diff
@@ -686,6 +719,32 @@ To start, pause, and stop the playback, follow the steps below:
   }
   ```
 
+<a name="control"></a>
+## Playing Mode
+
+It is possible to determine whether to request playback of PCM data synthesized by the TTS service or to receive and play it directly by the TTS client.
+
```

Comment:
It will be better to add the difference between `mode` and `playing mode`.
```suggestion
> [!NOTE]
> Playing mode is different from TTS mode. Playing mode only determines a subject to play PCM data. ...
```

Reply:
Thank you for adding this. I've added it.